src/machine/machined.h

```diff
@@ -57,7 +59,11 @@ int manager_unref_unit(Manager *m, const char *unit, sd_bus_error *error);
 int manager_unit_is_active(Manager *manager, const char *unit);
 int manager_job_is_active(Manager *manager, const char *path);
 
+#if ENABLE_NSCD
 int manager_enqueue_nscd_cache_flush(Manager *m);
+#else
+static inline void manager_enqueue_nscd_cache_flush(Manager *m) {}
```

Comment:
Will the compiler nag about \*m being unused with the flags we set in Meson?

Reply:
I don't think so. With gcc in F33 and rawhide, it doesn't. (And if it turns out it does with some combination of flags, we can address the issue then.)

Reply:
Great, thanks for checking!